<h2>Busso is failing</h2>

<pre><code>Traceback (most recent call last):

  File &#34;C:\Users\me\Dropbox\felix\test.py&#34;, line 17, in test_alert_email
    raise Exception(&#34;What the heck?&#34;)

Exception: What the heck?
</code></pre>
